addons/website_event/controllers/main.py

```diff
@@ -198,30 +179,28 @@ def _add_event(self, event_name=None, context={}, **kwargs):
             'date_end': (date_begin + timedelta(days=(1))).strftime('%Y-%m-%d'),
             'seats_available': 1000,
         }
-        event_id = Event.create(request.cr, request.uid, vals, context=context)
-        event = Event.browse(request.cr, request.uid, event_id, context=context)
+        event = request.env['event.event'].with_context(context).create(vals)
         return request.redirect("/event/%s/register?enable_editor=1" % slug(event))
 
     def get_formated_date(self, event):
         start_date = datetime.strptime(event.date_begin, tools.DEFAULT_SERVER_DATETIME_FORMAT).date()
         end_date = datetime.strptime(event.date_end, tools.DEFAULT_SERVER_DATETIME_FORMAT).date()
-        return ('%s %s%s') % (start_date.strftime("%b"), start_date.strftime("%e"), (end_date != start_date and ("-"+end_date.strftime("%e")) or ""))
+        return ('%s %s%s') % (start_date.strftime("%b"), start_date.strftime("%e"), (end_date != start_date and ("-" + end_date.strftime("%e")) or ""))
 
     @http.route('/event/get_country_event_list', type='http', auth='public', website=True)
     def get_country_events(self, **post):
-        cr, uid, context, event_ids = request.cr, request.uid, request.context, []
-        country_obj = request.registry['res.country']
-        event_obj = request.registry['event.event']
+        Country = request.env['res.country']
+        Event = request.env['event.event']
         country_code = request.session['geoip'].get('country_code')
         result = {'events': [], 'country': False}
         if country_code:
-            country_ids = country_obj.search(cr, uid, [('code', '=', country_code)], context=context)
-            event_ids = event_obj.search(cr, uid, ['|', ('address_id', '=', None), ('country_id.code', '=', country_code), ('date_begin', '>=', time.strftime('%Y-%m-%d 00:00:00')), ('state', '=', 'confirm')], order="date_begin", context=context)
+            country_ids = Country.search([('code', '=', country_code)])
+            event_ids = Event.search(['|', ('address_id', '=', None), ('country_id.code', '=', country_code), ('date_begin', '>=', time.strftime('%Y-%m-%d 00:00:00')), ('state', '=', 'confirm')], order="date_begin")
         if not event_ids:
-            event_ids = event_obj.search(cr, uid, [('date_begin', '>=', time.strftime('%Y-%m-%d 00:00:00')), ('state', '=', 'confirm')], order="date_begin", context=context)
-        for event in event_obj.browse(cr, uid, event_ids, context=context)[:6]:
+            event_ids = Event.search([('date_begin', '>=', time.strftime('%Y-%m-%d 00:00:00')), ('state', '=', 'confirm')], order="date_begin")
+        for event in event_ids:
             if country_code and event.country_id.code == country_code:
-                result['country'] = country_obj.browse(cr, uid, country_ids[0], context=context)
+                result['country'] = Country.browse(country_ids[0].id)
```

Comment:
This code does not make sens. You already have browse records du to the search hereabove. Do not simply translate each line one line at a time, you also have to see the function flow and see how it is impacted by the new API. Please clean the whole method that does unnecessary search / browse.